pushapkscript/test/test_googleplay.py

```diff
@@ -16,179 +16,97 @@
 
     def test_craft_push_config_allows_to_contact_google_play_or_not(self, mock_push_apk):
-        publish_to_googleplay(self.task_payload, self.products['aurora'], self.apks, contact_google_play=True)
+        publish_to_googleplay({}, {}, self.publish_config, self.apks, contact_google_play=True)
         _, args = mock_push_apk.call_args
         assert args['contact_google_play'] is True
 
-        publish_to_googleplay(self.task_payload, self.products['aurora'], self.apks, False)
+        publish_to_googleplay({}, {}, self.publish_config, self.apks, False)
         _, args = mock_push_apk.call_args
         assert args['contact_google_play'] is False
 
     def test_craft_push_config_skip_checking_multiple_locales(self, mock_push_apk):
         product_config = {
-            'has_nightly_track': False,
-            'service_account': 'product',
-            'certificate': '/path/to/product.p12',
-            'skip_check_package_names': True,
             'skip_check_multiple_locales': True,
         }
-        publish_to_googleplay(self.task_payload, product_config, self.apks, contact_google_play=True)
+        publish_to_googleplay({}, product_config, self.publish_config, self.apks, contact_google_play=True)
         _, args = mock_push_apk.call_args
         assert args['skip_check_multiple_locales'] is True
 
     def test_craft_push_config_skip_checking_same_locales(self, mock_push_apk):
         product_config = {
-            'has_nightly_track': False,
-            'service_account': 'product',
-            'certificate': '/path/to/product.p12',
-            'skip_check_package_names': True,
             'skip_check_same_locales': True,
         }
-        publish_to_googleplay(self.task_payload, product_config, self.apks, contact_google_play=True)
+        publish_to_googleplay({}, product_config, self.publish_config, self.apks, contact_google_play=True)
         _, args = mock_push_apk.call_args
         assert args['skip_check_same_locales'] is True
 
     def test_craft_push_config_expect_package_names(self, mock_push_apk):
-        product_config = {
-            'has_nightly_track': False,
-            'service_account': 'product',
-            'certificate': '/path/to/product.p12',
-            'expected_package_names': ['org.mozilla.focus', 'org.mozilla.klar']
+        publish_config = {
+            'google_play_track': 'beta',
+            'package_names': ['a', 'b'],
+            'service_account': 'service_account',
+            'google_credentials_file': '/google_credentials.p12',
         }
-        publish_to_googleplay(self.task_payload, product_config, self.apks, contact_google_play=True)
+        publish_to_googleplay({}, {}, publish_config, self.apks, contact_google_play=True)
         _, args = mock_push_apk.call_args
-        assert args['expected_package_names'] == ['org.mozilla.focus', 'org.mozilla.klar']
+        assert args['expected_package_names'] == ['a', 'b']
```

Comment:
Nit: Let's make `a` and `b` looks like they are package_names IDs. I'm fine changing `org.mozilla.focus` into something that is less mozilla-specific, but I'm afraid `a` is too generic and may introduce mental overhead if more tests apply this pattern. E.g.:

> What is `a` in this particular test?
 
> Oh, that's the name of a custom Google Play track.